cg/cli/upload/clinical_delivery.py

```diff
@@ -104,14 +105,24 @@ def auto_fastq(context: click.Context, dry_run: bool):
                 )
                 analysis_obj.uploaded_at = dt.datetime.now()
             else:
-                LOG.warning(
+                LOG.debug(
                     "Upload to clinical-delivery for %s has already started, skipping",
                     analysis_obj.family.internal_id,
                 )
             continue
         case: Family = analysis_obj.family
         LOG.info("Uploading family: %s", case.internal_id)
         analysis_obj.upload_started_at = dt.datetime.now()
-        context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
+        try:
+            context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
+        except Exception as error:
+            LOG.error(f"Upload of case {case.internal_id} failed")
+            LOG.error(error)
+            exit_code: int = EXIT_FAIL
+            continue
```

Comment:
Would we want some more explicit errors to catch?

Reply:
I don't think we want to. I see no type of error where we would not want to try with the next case 🤔  Do you see any potential bugs here?

Reply:
Not really, I was thinking that maybe it is unnecessary to continue (and that an error should be raised) if we get a connection dropped error or something. But feel free to keep it as is.